Cargo.toml

```diff
@@ -40,6 +43,8 @@ mint = { version = "0.5", optional = true, default-features = false }
 num-traits = { version = "0.2.14", optional = true, default-features = false }
 rand = { version = "0.8", optional = true, default-features = false }
 serde = { version = "1.0", optional = true, features = ["derive"] }
+rkyv = {version = "0.7.4", optional = true, default-features = false }
```

Comment:
I'd recommend changing this version dependency to `0.7`. The only changes going in 0.7.x will not be breaking.